platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceImpl.java

```diff
@@ -88,47 +91,53 @@
      */
     public ApplicationServiceImpl(List<BundleStateService> bundleStateServices) {
         BundleContext context = getContext();
-        ServiceReference<FeaturesService> featuresServiceRef = context
-                .getServiceReference(FeaturesService.class);
+        ServiceReference<FeaturesService> featuresServiceRef = context.getServiceReference(
+                FeaturesService.class);
         this.featuresService = context.getService(featuresServiceRef);
         this.bundleStateServices = bundleStateServices;
-        ignoredApplicationNames = new HashSet<String>();
+        ignoredApplicationNames = new HashSet<>();
 
     }
 
     @Override
     public Set<Application> getApplications() {
         logger.trace("Getting all applications.");
-        Repository[] repos = featuresService.listRepositories();
-        logger.debug("Found {} applications from feature service.", repos.length);
+        Repository[] repos = {};
+        try {
+            repos = featuresService.listRepositories();
+
+            logger.debug("Found {} applications from feature service.", repos.length);
 
-        if (logger.isDebugEnabled()) {
-            for (int ii = 0; ii < repos.length; ++ii) {
-                logger.debug("Repo/App {}: {}", ii, repos[ii].getName());
+            if (logger.isDebugEnabled()) {
+                for (int ii = 0; ii < repos.length; ++ii) {
+                    logger.debug("Repo/App {}: {}", ii, repos[ii].getName());
+                }
             }
+        } catch (Exception e) {
```

Comment:
Should we catch `RuntimeException` instead? That way, if a new checked exception gets thrown in the try block in the future that we need to handle, we know about it?